### DISCUSSION TITLE
IP allowed web page

### DISCUSSION THREAD

**Discussion by PanamaGit** [wg-easy/wg-easy#515] [Q&A] [answered]
Good afternoon.
How can I restrict the wireguard web page by the allowed ip? wg-easy is installed in docker

**Comment by eandersons** [comment, marked as answer]
It depends on your setup. If you are using a proxy server for WireGuard Easy user interface, then IP restrictions can be defined there.
Another approach would be to add a firewall rule.

A side note that may or may not be relevant - if the port mapping is specified without an explicit IP address, it is published on 0.0.0.0, meaning that the mapped port is accessible to everyone in network that can access the machine (of course, if firewall lets incoming requests to this port through). The port mapping `127.0.0.1:51821:51821` makes the port accessible only from the machine itself, therefore a proxy is needed to access it from outside.
